pom.xml

```diff
@@ -2474,6 +2474,7 @@
                                     -Xep:MissingSummary:OFF \
                                     -Xep:MutablePublicArray:ERROR \
                                     -Xep:NarrowingCompoundAssignment:ERROR \
+                                    -Xep:NullableOptional:ERROR \
                                     -Xep:NullOptional:ERROR \
```

Comment:
is the sorting case-sensitive or case-insensitive?
for example, both vim and Intellij's "Sort lines" sort these lines differently

Reply:
I was wondering about it too, but I always find it confusing, so I usually don't bother. Reordering is not a problem. We can definitely decide that "whatever the tools are doing" is the policy here :)

Reply:
Reordered